packages/w3up-client/src/client.js

```diff
@@ -282,6 +282,76 @@ export class Client extends Base {
     return space
   }
 
+  /**
+   * Share an existing space with another Storacha account via email address delegate.
+   * Delegates access to the space to the specified email account with the following permissions:
+   * - space/* - for uploading blobs
+   * - upload/*- for registering uploads
+   * - filecoin/* - for submitting to the filecoin pipeline
+   * - access/* - for re-delegating access to other devices
+   * - usage/* - for querying usage
+   * The default expiration is set to infinity.
+   *
+   * @typedef {Omit<import('./types.js').UCANOptions, 'audience'> & { audienceMeta?: import('./types.js').AgentMeta }} ShareOptions
+   *
+   * @param {import("./types.js").EmailAddress} delegateEmail - Email of the account to share the space with.
+   * @param {import('./types.js').SpaceDID} spaceDID - The DID of the space to share.
+   * @param {import('./types.js').ServiceAbility[]} abilities - Abilities to delegate to the delegate account.
```

Comment:
I think we can move the `abilities` to the options because it is optional. But for `email` and `spaceDID` it is nice to have them as part of the function arguments because they are required. WDYT?

Reply:
Yes for sure - apologies I meant only `abilities`.